doc/getting-started.md

```diff
@@ -25,6 +26,12 @@ This guide will help you get started writing contracts with CTL. Please also see
 
 ## Prerequisites
 
+### Setting up Nix caches
+
+Nix caches improve build speeds by allowing to download pre-built dependencies instead of compiling them from source.
+
+Follow [these instructions](https://github.com/input-output-hk/cardano-ledger#nix-cache) to add the caches.
```

Comment:
As far as I understand, without MLabs private caches there will be still many builds happening.

Reply:
I don't think so, `Push+pull caches` are set to `public-plutonomicon` in CI config.

Reply:
Obviously, it does not guarantee that we don't need to use mlabs cachix

Reply:
I added public-plutonomicon cache

Reply:
I asked what to do with mlabs cachix internally